data/core/units/monsters/Wolf_Great.cfg

```diff
@@ -16,7 +16,7 @@
     advances_to=null
     cost=21
     usage=scout
-    description=_ "Wolves are aggressive animals that hunt in packs. Although weak individually, a wolf pack can kill even the strongest man in minutes."
+    description=_ "The greatest of wolves may almost possess the strength of a direwolf, and they should never be underestimated, for those who have have suffered terrible fates."
```

Comment:
I can see why you used 2x "have" words here. maybe a comma in between?

Reply:
I just removed it. I sounds better. Thank you for pointing this out.